model.py

```diff
@@ -36,6 +36,6 @@ def connect_to_db(app,
 if __name__ == "__main__":
 
     from server import app
-    connect_to_db(app)
+    connect_to_db(app,db_uri="postgresql:///newb")
```

Comment:
are these changes to model intended? If so, how will the changes impact other contributors?

Reply:
This change wasn't intended.  Sorry.  Are you able to reject it or do I need to do something on my end to undo it?

Reply:
you can push to the same branch again with the changes :)